src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -230,7 +230,7 @@ openssl x509 -pubkey -noout -in bitbucket_publickey.cer  > bitbucket_publickey.p
 
 ##### III. Connect your Bitbucket Server application with Sentry
 {% capture __alert_content -%}
-Make sure you have whitelisted [Sentry's IP ranges.](https://docs.sentry.io/ip-ranges/)
+Make sure you have whitelisted [Sentry's IP ranges]({%- link _documentation/meta/ip-ranges.md -%}).
```

Comment:
Nit: change "Make sure you have..." to "Confirm [Sentry's IP ranges] ({%- link _documentation/meta/ip-ranges.md -%}) are whitelisted" 

Or, more directive "Whitelist [Sentry's IP ranges] ({%- link _documentation/meta/ip-ranges.md -%})"